src/github.com/getlantern/elevate/command_windows.go

```diff
@@ -0,0 +1,81 @@
+package elevate
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+)
+
+var (
+	scriptFile *os.File
+	scriptPath string
+)
+
+func init() {
```

Comment:
Would be better to initialise it lazily. This package might be imported but not have chance to actually run.

Having an `init()` which may panic might not be a good idea in general.